lib/backend/syncersv1/updateprocessors/rules.go

```diff
@@ -116,6 +133,20 @@ func RuleAPIV2ToBackend(ar apiv2.Rule, ns string) model.Rule {
 	}
 }
 
+// parseNamespaceSelector takes a v2 namespace selector and returns the appropriate v1 representation
+// by prefix the keys with the `pcns.` prefix. For example, `k == 'v'` becomes `pcns.k == 'v'`.
```

Comment:
prefix (ing)

Reply:
isn't it better to prefix with "pcns/" (i.e. `/` rather than `.`) - that means we don't have to limit labels to not include the `pcns.` prefix.

Reply:
I discussed with @lxpollitt and decided to leave it as-is.

We'd still need to prevent people from labeling with `pcns/` (labels allow for the `/` character).